packages/grid/data-grid/src/tests/filtering.DataGrid.test.tsx

```diff
@@ -251,6 +251,110 @@ describe('<DataGrid /> - Filter', () => {
       expect(getColumnValues()).to.deep.equal(['Nike']);
     });
 
+    it('should allow operator isEmpty for number field', () => {
+      const rows = [
+        {
+          id: 0,
+          quantity: 0,
+        },
+        {
+          id: 1,
+          quantity: null,
+        },
+        {
+          id: 2,
+          quantity: 100,
+        },
+      ];
+      render(
+        <TestCase
+          operatorValue="isEmpty"
+          field="quantity"
+          columns={[{ field: 'id' }, { field: 'quantity', type: 'number' }]}
+          rows={rows}
+        />,
+      );
+      expect(getColumnValues(0)).to.deep.equal(['1']);
+    });
+
+    it('should allow operator isNotEmpty for number field', () => {
+      const rows = [
+        {
+          id: 0,
+          quantity: 0,
+        },
+        {
+          id: 1,
+          quantity: null,
+        },
+        {
+          id: 2,
+          quantity: 100,
+        },
+      ];
+      render(
+        <TestCase
+          operatorValue="isNotEmpty"
+          field="quantity"
+          columns={[{ field: 'id' }, { field: 'quantity', type: 'number' }]}
+          rows={rows}
+        />,
+      );
+      expect(getColumnValues(0)).to.deep.equal(['0', '2']);
+    });
+
+    it('should allow operator isEmpty for date field', () => {
```

Comment:
There's a "date operators" group only for tests related to the operators available in the `date` column type.